cookbooks/aws-parallelcluster-config/resources/fetch_config.rb

```diff
@@ -28,6 +29,14 @@
 end
 
 action_class do # rubocop:disable Metrics/BlockLength
+  def execute_command(label, command)
+    execute label do
+      command command
```

Comment:
Are you sure the parameter `command` does not shadow the keyword `command`?

Reply:
yes, verified it's not shadowed. Nonetheless I have changed the name for clarity. Thanks